bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieSideConnectionPeer.java

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.bookie;
+
+import java.net.SocketAddress;
+import java.util.Collection;
+import org.apache.bookkeeper.auth.BookKeeperPrincipal;
+
+/**
+ * Represents the connection to a BookKeeper client, from the Bookie side 
+ */
+public interface BookieSideConnectionPeer {
```

Comment:
Two comments:

- it seems that BookieSideConnectionPeer has exactly same interface with ClientSideConnectionPeer. Can we have one common parent class.
- Can we remove 'Side' from the name?

How about have the common interface 'ConnectionPeer' and have 'ServerConnectionPeer' and 'ClientConnectionPeer' extend from the parent interface?